src/main/java/com/autotune/common/experiments/ContainerAmendData.java

```diff
@@ -0,0 +1,84 @@
+/*******************************************************************************
+ * Copyright (c) 2021, 2022 Red Hat, IBM Corporation and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package com.autotune.common.experiments;
+
+import com.google.gson.annotations.SerializedName;
+import io.fabric8.kubernetes.api.model.EnvVar;
+import io.fabric8.kubernetes.api.model.Quantity;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * todo
+ */
+
+public class ContainerAmendData {
+    @SerializedName("requests")
+    private Map<String, Quantity> requestPropertiesMap = new HashMap<String, Quantity>();
+    @SerializedName("limits")
+    private Map<String, Quantity> LimitPropertiesMap = new HashMap<String, Quantity>();
+    @SerializedName("env")
+    private List<EnvVar> envList = new ArrayList<EnvVar>();
+    private String trailNumber;
+
+    public ContainerAmendData() {
+    }
+
+    public Map<String, Quantity> getRequestPropertiesMap() {
+        return requestPropertiesMap;
+    }
+
+    public void setRequestPropertiesMap(Map<String, Quantity> requestPropertiesMap) {
+        this.requestPropertiesMap = requestPropertiesMap;
+    }
+
+    public Map<String, Quantity> getLimitPropertiesMap() {
+        return LimitPropertiesMap;
+    }
+
+    public void setLimitPropertiesMap(Map<String, Quantity> limitPropertiesMap) {
+        LimitPropertiesMap = limitPropertiesMap;
+    }
+
+    public List<EnvVar> getEnvList() {
+        return envList;
+    }
+
+    public void setEnvList(List<EnvVar> envList) {
+        this.envList = envList;
+    }
+
+    public String getTrailNumber() {
```

Comment:
Trial instead of Trail, same in the API below as well. I suggest you do a search and replace of Trail in your codebase.